src/Plugin.php

```diff
@@ -798,6 +799,14 @@ public function uninstall($ID)
 
             $type = INFO;
             $message = sprintf(__('Plugin %1$s has been uninstalled!'), $this->fields['name']);
+
+            Event::log(
+                '',
+                Plugin::class,
+                4,
```

Comment:
Level should probably be 3.

Maybe you can introduce some constants to make log levels values more readable.

```suggestion
                3,
```